### PR TITLE
Support rich text

### PR DESCRIPTION
**Pull request by LesterLyu**
## Support rich text
You can read/write rich text to cells.
```js
const cell = workbook.sheet(0).cell('A1');

// transform exsisting string cell to rich text
cell.toRichText();

// reset a rich text cell
cell.value('string');
cell.value(new Date());

// create new cell with rich text
workbook.sheet(0).cell('B1').toRichText()
    .add('test', { bold: true, italic: true })
    .add('123', { italic: true, fontColor: '123456' });

// modify a rich text cell
cell.value().get(0).style('fontFamily', 'Calibri')
cell.toRichText().get(0).style('fontFamily', 'Calibri')

// read multiple styles
cell.toRichText().get(0).style(['fontFamily', 'italic', 'bold'])
```

### CODE REVIEW

**Comment by eddiecorrigall** [comment]
Thank you for your hard work. I'll review and see if I can help get this passing.

File: lib/Cell.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ const regexify = require("./regexify");
 const xmlq = require("./xmlq");
 const FormulaError = require("./FormulaError");
 const Style = require("./Style");
+const RichTexts = require("./RichTexts");
```
Comment (eddiecorrigall) [inline lib/Cell.js]: I think you mean:
`const RichText = require("./RichText");`

**Comment by LesterLyu** [comment]
Please change anything you want, I think RichTexts class need more helper methods, such as for each.

**Comment by Fauny** [comment]
line-break in richtext does not work properly.
```js
cell.add('a\nb', {bold: true}); // this works fine.

// these can neither work:
cell.add('a\n', {bold: true});
cell.add('b\n', {bold: false});
cell.add('c', {bold: true});

// nor these:
cell.add('a', {bold: true});
cell.add('\nb', {bold: false});
cell.add('\nc', {bold: true});
```

**Comment by LesterLyu** [comment]
> line-break in richtext does not work properly.
> 
> ```js
> cell.add('a\nb', {bold: true}); // this works fine.
> 
> // these can neither work:
> cell.add('a\n', {bold: true});
> cell.add('b\n', {bold: false});
> cell.add('c', {bold: true});
> 
> // nor these:
> cell.add('a', {bold: true});
> cell.add('\nb', {bold: false});
> cell.add('\nc', {bold: true});
> ```

Can you please try ```\r\n``` line separator to see if it works?

**Comment by Fauny** [comment]
> Can you please try `\r\n` line separator to see if it works?

I tried this and still can't work.

**Comment by LesterLyu** [comment]
> > Can you please try `\r\n` line separator to see if it works?
> 
> I tried this, still can't work.

Try to set ```cell.style('wrapText', true)```
https://stackoverflow.com/questions/8613850/c-sharp-excel-interop-put-a-line-break-in-the-text-of-a-cell-in-a-excel

**Comment by LesterLyu** [comment]
I will make a commit to convert ```'\n'``` to ```'\r\n'``` and automatic set ```cell.style('wrapText', true)``` if line separator is detected.

**Comment by Fauny** [comment]
> > > Can you please try `\r\n` line separator to see if it works?
> > 
> > 
> > I tried this, still can't work.
> 
> Try to set `cell.style('wrapText', true)`
> https://stackoverflow.com/questions/8613850/c-sharp-excel-interop-put-a-line-break-in-the-text-of-a-cell-in-a-excel

I'v set wrapText = true, and tried both \n and \r\n, still can't work.

**Comment by LesterLyu** [comment]
@Fauny @eddiecorrigall please see #190

**Pull request closed without merging**